File: CSharp/WcfService1/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

namespace TestAPIns
{
    [DataContract]
    public class Person
    {
        [DataMember(Name="id")]
        public int ID;
        [DataMember(Name="name")]
        public string Name;
        [DataMember(Name="age")]
        public string Age;
        [DataMember(Name="bookmarks")]
        public List<Bookmark> Bookmarks;

    } 
}

File: ruby_sinatra/models/person.rb
class Person 
  include MyJson

  attr_accessor :id, :name, :age, :bookmarks

  def initialize()
    @bookmarks = Array.new
  end

  def initialize(name, age)
    @name = name
    @age = age
    @bookmarks = Array.new
  end


#  property :bookmarks,  Bookmark
end
